src/dpo/src/detailed_manager.cxx

```diff
@@ -85,7 +85,7 @@ DetailedMgr::DetailedMgr(Architecture* arch,
   targetUt_ = 1.0;
 
   // For generating a move list...
-  moveLimit_ = 10;
+  moveLimit_ = 100;
```

Comment:
What is the implication of this change?  How does it relate to 1-site gaps?

Reply:
assume there are 100 consequetive cells on the left and then a one-site gap followed by another 100 touching cells. We have to shift either of the 100 cells to the left or to the right.

If the move limit was 10, test cases like ibex failed because it needed to move more than 10 cells sometimes. If the move limit is small, it sees that the shift would result in an overlap so it will skip this move, which would result in a one-site gap violation that we didn't fix. 

Relaxing this to a 100 doesn't affect anything that doesn't need to move all of these cells. It only allows bigger test cases to fail less because the probability of having 100 consequetive cells is lower than having 10.

Reply:
We discussed having a larger threshold specifically for one-site